modules/backing-up-red-hat-quay.adoc

```diff
@@ -96,11 +136,16 @@ Example output:
 ----
 registry-quay-config-editor-77847fc4f5-nsbbv   1/1     Running            0          9m1s
 registry-quay-database-66969cd859-n2ssm        1/1     Running            0          6d1h
-registry-quay-mirror-758fc68ff7-5wxlp          1/1     Running            0          8m29s
-registry-quay-mirror-758fc68ff7-lbl82          1/1     Running            0          8m29s
 registry-quay-redis-7cc5f6c977-956g8           1/1     Running            0          5d21h
 ----
```

Comment:
The output in 3.7 is 
```
$ oc get pod 
NAME                                               READY   STATUS      RESTARTS      AGE
quay-operator.v3.7.1-6f9d859bd-p5ftc               1/1     Running     0             12m
quayregistry-clair-postgres-7487f5bd86-xnxpr       1/1     Running     1 (12m ago)   12m
quayregistry-quay-app-upgrade-xq2v6                0/1     Completed   0             12m
quayregistry-quay-config-editor-6dfdcfc44f-hlvwm   1/1     Running     0             73s
quayregistry-quay-database-859d5445ff-cqthr        1/1     Running     0             12m
quayregistry-quay-redis-84f888776f-hhgms           1/1     Running     0             12m
```
The output in 3.4 - 3.6 is 
```
NAME                                              READY   STATUS      RESTARTS   AGE
quayregistry-clair-postgres-7b5bd874d6-7dzdv      1/1     Running     0          26m
quayregistry-quay-app-upgrade-lz8d9               0/1     Completed   0          26m
quayregistry-quay-config-editor-d9fb9b9fb-lpjk4   1/1     Running     0          26m
quayregistry-quay-database-74c98466c6-nvqsn       1/1     Running     0          26m
quayregistry-quay-postgres-init-4h6xt      	  0/1     Completed   0          26m
quayregistry-quay-redis-5cf66f79b5-c8tpr          1/1     Running     0          26m
```
Due to the scale down method is different between 3.7 and 3.6-,  the output is different too.  Do we really need a example here? or just highlight  
```
`registry-quay-app`, `registry-quay-mirror` and `registry-clair-app` pods (depending on which components you set to be managed by operator) to disappear.`
```

Reply:
probably not, but it's in line with the rest of the docs style